packages/web-components/fast-components/docs/design/match-media-stylesheets.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: match-media-stylesheets
 title: MatchMedia Stylesheets
-custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-components/docs/guide/styling-components.doc.md
+custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-components/docs/design/styling-components.md
```

Comment:
this custom edit url is `styling-components` but should it be `match-media-stylesheets`?

Reply:
Doh. Copy/Paste error. Good catch.